salesforce/10.9/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -0,0 +1,5130 @@
+| Object TTL a| Number |  Specifies the amount of time an object in the Connections pool lives if it was not used. |  30 |
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Object TTL |  SECONDS |
+| Max Entries a| Number |  Specifies the amount of objects that live in the pool at a specific moment. When this number is reached and a new object is needed, no new object will be created and the application waits until a connection is released. |  10 |
+| Username a| String |  Username used to initialize the session. |  | x
+| Password a| String |  Password used to authenticate the user. |  | x
+| Security Token a| String |  User's security token. It can be omitted if your IP has been white listed on Salesforce. |  |
+| Authorization URL a| String |  Web service URL responsible for user authentication. This is the URL for the endpoint that is configured to handle SOAP authentication requests. Defaults to the URL containing the latest api version supported by the connector. For example: `+https://login.salesforce.com/services/Soap/u/48.0+`|  |
+| Session Id a| String |  Value that identifies an active Salesforce session |  |
+| Service Endpoint a| String |  Specifies the service endpoint. This value will only be used if the sessionId configuration property is used. |  |
+| TLS configuration a| <<Tls>> | Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol then the user needs to configure at least the keystore in the `tls:context` child element of the `listener-config`. |  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+
+[[sfdc-config_config-with-oauth]]
+=== OAuth v2.0
+
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Connection Timeout a| Number |  Specifies the amount of time that the consumer attempts to establish a connection before it times out. Works in tandem with Time unit. Values less than one millisecond are converted to 0. |  0 |
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use for the Connection Timeout parameter |  SECONDS |
+| Login Request Timeout a| Number | Amount of time, in the unit defined in the Login Request Time unit field, for which the consumer waits for the login request to be received before the request times out. This value works in with the Login Request Time unit field value. Values less than one millisecond are converted to 0. | |
```

Comment:
```suggestion
| Login Request Timeout a| Number | Amount of time, in the unit defined in the Login Request Time unit field, that the consumer waits for the login request to be received before the request times out. This value works in conjunction with the Login Request Time unit field value. Values less than one millisecond are converted to 0. | |
```

Reply:
I don't even think this sentence is necessary since you already mentioned the field in the first sentence. I would delete:
This value works in conjunction with the Login Request Time unit field value.